tutor/src/screens/question-library/exercises-display.jsx

```diff
@@ -389,7 +415,7 @@ class ExercisesDisplay extends React.Component {
     return (
       <StyledExerciseDisplay>
         <StyledCourseBreadcrumb course={course} currentTitle="Question Library" />
-        <div className="controls-wrapper">
+        <div className={cn('controls-wrapper', { 'isSticky': this.isControlsSticky })}>
           <ExerciseControls
```

Comment:
css classNames should use lowercase & dashes, not camelCase, I also kinda prefer declarative names for them instead of `is-`, so this could be just `sticky-controls` maybe?